lib/ActivityStreams.js

```diff
@@ -129,6 +132,18 @@ ActivityStreams.prototype = {
   },
 
   /**
+   * Sets up the toolbar button that links to the Timeline page.
+   */
+  _setupButton() {
+    ActionButton({
+      id: "activity-streams-link",
+      label: "Timeline",
+      icon: data.url("content/img/list-icon.svg"),
+      onClick: () => tabs.open("about:timeline")
```

Comment:
Can we point to the `resource://activity-streams....` link until this is implemented?